handlers/freshchat/freshchat.go

```diff
@@ -0,0 +1,284 @@
+package freshchat
+
+/*
+ * Handler for FreshChat
+ */
+import (
+	"bytes"
+	"context"
+	"crypto"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/nyaruka/courier"
+	"github.com/nyaruka/courier/handlers"
+	"github.com/nyaruka/courier/utils"
+	"github.com/nyaruka/gocommon/urns"
+)
+
+const (
+	configFCWebhookKey = "secret"
+	configFCAgentID    = "username"
```

Comment:
No need to use `FC` here, it is a private variable in the freshchat module. (this is a standard golang wart)

I don't care that much, but if you are creating new config vars here may as well name them the same. (IE, `webhook_key`) (if alternatively you are trying to reuse the constant then set it to the constant value, ie `courier.ConfigSecret`)

Reply:
yeah i was in the middle of switching it back- creating new config vars had a domino effect in other areas (tests, etc) that didn't seem worth the effort.